docs/gettingstarted/index.rst

```diff
@@ -81,11 +81,8 @@ Getting started with Flocker
 
          Install the Flocker client on your Mac (requires Homebrew):
 
-         .. version-code-block:: console
-
-            you@laptop:~$ brew update && \
-              brew tap clusterhq/flocker && \
-              brew install flocker-|latest-installable|
+         .. task:: test_homebrew flocker-|latest-installable|
+            :prompt: you@laptop:~$
```

Comment:
I'm happy with it, and as you say this is tested so if it stops working we can swap them or change things.